python/paddle/v2/parameters.py

```diff
@@ -138,14 +129,27 @@ def __getitem__(self, key):
                     each_gradient_machine, key)
                 # for simplify implementation now, we always copy from C++
                 assert isinstance(param, api.Parameter)
-                val = param.getBuf(api.PARAMETER_VALUE)
+                val = param.getBuf(param_type)
                 assert isinstance(val, api.Vector)
                 val = val.copyToNumpyArray()
                 return val
                 # else continue
 
             raise RuntimeError("Unexpected branch")
 
+    def __getitem__(self, key):
```

Comment:
`__getitem__` will called when doing `param[k]`, it's an operator reload. Need to keep this the same as before.